api/v1beta1/hyperconverged_types.go

```diff
@@ -25,6 +25,18 @@ const (
 	HyperConvergedUninstallStrategyBlockUninstallIfWorkloadsExist HyperConvergedUninstallStrategy = "BlockUninstallIfWorkloadsExist"
 )
```

Comment:
Do we need only the name of the policy or also the capability to fine tune predefined policies?
If just the name (which in the beginning could only be "ConfigMap" pointing to a configmap with a fixed name in the operator namespace), why not:

```
type HyperConvergedTuningPolicy string
...
HyperConvergedTuningPolicyConfigMap                HyperConvergedTuningPolicy = "ConfigMap"
...
// +kubebuilder:validation:Enum=ConfigMap
// +optional
TuningPolicy HyperConvergedTuningPolicy `json:"tuningPolicy,omitempty"`
```

Reply:
I agree, I have used this approach. I think it's much more cleaner now.